.github/workflows/codeql-analysis.yml

```diff
@@ -56,10 +59,18 @@ jobs:
         # Prefix the list here with "+" to use these queries and those in the config file.
         # queries: ./path/to/local/query, your-org/your-repo/queries@main
 
+    - name: Setup .NET Core SDK
+      uses: actions/setup-dotnet@v3.0.3
+      with:
+        dotnet-version: ${{ env.dotnet_sdk_version }}
+
+    - name: Build
+      run: dotnet build
+
     # Autobuild attempts to build any compiled languages  (C/C++, C#, or Java).
     # If this step fails, then you should remove it and run the build manually (see below)
-    - name: Autobuild
-      uses: github/codeql-action/autobuild@v2
+    #- name: Autobuild
```

Comment:
Hasn't CodeQL been working well recently? Doesn't commenting this out disable it?

Reply:
> Doesn't commenting this out disable it?

No, it just disables the Autobuild step which fails and does nothing but building, which we can well do manually.
See the step above, where I enabled building manually.

Reply:
OK :+1: